model/project_ref.go

```diff
@@ -1141,18 +1141,24 @@ func GetTasksWithOptions(projectName string, taskName string, opts GetProjectTas
 }
 
 func FindFirstProjectRef() (*ProjectRef, error) {
-	projectRef := &ProjectRef{}
+	projectRefSlice := &[]ProjectRef{}
 	pipeline := projectRefPipelineForValueIsBool(ProjectRefPrivateKey, RepoRefPrivateKey, false)
-	pipeline = append(pipeline, bson.M{"$sort": "-" + ProjectRefDisplayNameKey}, bson.M{"$limit": 1})
+	pipeline = append(pipeline, bson.M{"$sort": bson.M{ProjectRefPrivateKey: -1}}, bson.M{"$limit": 1})
 	err := db.Aggregate(
 		ProjectRefCollection,
 		pipeline,
-		projectRef,
+		projectRefSlice,
```

Comment:
would it work for `projectRefSlice` to not be a pointer and the pass in a pointer to it here?

Reply:
Yeah I got rid of a lot of the pointer usage. Go's similarity to C made me weary of things like taking the address of a "stack variable" but the Go docs tell me that is ok.